automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala

```diff
@@ -73,9 +73,12 @@ class RawlsApiSpec extends FreeSpec with Matchers with CleanUp with BillingFixtu
         val userBWorkspace = Rawls.workspaces.list()(studentBToken)
         userBWorkspace should include(workspaceNameB)
 
+        Orchestration.billing.removeUserFromBillingProject(projectName, studentA.email, Orchestration.billing.BillingProjectRole.User)(ownerAuthToken)
+        Orchestration.billing.removeUserFromBillingProject(projectName, studentB.email, Orchestration.billing.BillingProjectRole.User)(ownerAuthToken)
```

Comment:
not for this PR, but the fact that this was missing before makes me wonder if `addUserToBillingProject` should register its own cleanup

Reply:
good point. `withCleanBillingProject` takes List(emails) parameter that automatically add and remove users from new proj.

Reply:
@helgridly i had it registering cleanups as a first iteration but the timing of things was strange. the cleanUps execute after the gpalloc'd project is released so it's not able to actually remove the users. i'll note a TODO with this information

@aweng98 i'd use that but it automatically adds them as project Owners which i'm not sure is appropriate for this test

Reply:
oh blergh. there might be potential to attach cleanup to the project-object that joel made recently, but this definitely Requires Some Thought

Reply:
@MatthewBemis You need to insert a `withCleanUp` in the fixtures to control when the registered clean-ups are run. I can help with that.